docs/en/concepts-and-designs/lal.md

```diff
@@ -240,6 +240,57 @@ metrics:
     exp: http_response_time.sum(['le', 'service', 'instance']).increase('PT5M').histogram().histogram_percentile([50,70,90,99])
 ```
 
+### SlowSql
```

Comment:
Is this original log to be sinked or abandoned? Or does it depend on further LAL? If so, I think we should abandon this logs from persistence in the example with comments to explain about why.